src/Symfony/Component/Translation/Dumper/FileDumper.php

```diff
@@ -109,6 +109,8 @@ protected function formatCatalogue(MessageCatalogue $messages, $domain, array $o
      * @param string           $domain
      *
      * @return string representation
+     *
+     * @deprecated since version 2.8, to be removed in 3.0. Use formatCatalogue() instead.
```

Comment:
and you should trigger a deprecation warning in the ``formatCatalogue`` of this class, telling it should be overwritten instead of overwriting ``format``